packages/docusaurus-theme-classic/src/theme/CodeBlock/index.tsx

```diff
@@ -152,26 +159,35 @@ export default function CodeBlock({
 
             <button
               type="button"
-              aria-label={translate({
-                id: 'theme.CodeBlock.copyButtonAriaLabel',
-                message: 'Copy code to clipboard',
-                description: 'The ARIA label for copy code blocks button',
-              })}
-              className={clsx(styles.copyButton, 'clean-btn')}
-              onClick={handleCopyCode}>
-              {showCopied ? (
-                <Translate
-                  id="theme.CodeBlock.copied"
-                  description="The copied button label on code blocks">
-                  Copied
-                </Translate>
-              ) : (
-                <Translate
-                  id="theme.CodeBlock.copy"
```

Comment:
CI is reporting that this id should be cleaned from i18n translations

Reply:
Yes, I just have not yet found a way to use the `title` and `aria-label` together so that it  avoid screen readers to double speak **after** change values of these attributes. I'll check it again, maybe find a solution this time.